frontend/src/pages/user.js

```diff
@@ -295,7 +293,7 @@ function Albums(props) {
     function openAlbum(props) { setMode("openedAlbum") };
     function setImagesProp(images) { setImages(images) };
 
-    const [modalShow, setModalShow] = React.useState(false);
+    //const [modalShow, setModalShow] = React.useState(false);
```

Comment:
If this line is unnecessary, delete it rather than commentating it out or add an additional comment explaining why this is being kept.